src/types/ynab/data/account.d.ts

```diff
@@ -3,7 +3,7 @@ import { YNABTransaction } from './transaction';
 
 export interface YNABAccount {
   accountName: string;
-  getAccountType(): YNABAccountType;
-  getAccountCalculation(): YNABAccountCalculation;
+  accountType: YNABAccountType;
+  accountCalculation: YNABAccountCalculation;
```

Comment:
Should these be marked as getters? (`get accountCalculation(): YNABAccountCalculation` et al)